.github/templates/android_common.yml.j2

```diff
@@ -0,0 +1,46 @@
+{% import 'common.yml.j2' as common %}
+
+{%- macro pytorch_android_gradle_custom_build_single(build_lite_interpreter) -%}
```

Comment:
If we're not expecting to have to re-use this in other workflows / multiple times in the same workflow I'd actually recommend against splitting it out into its own file / jinja2 macro

For a lot of these it's already complex enough to have to contribute to these workflows let alone having to track down multiple changes in multiple files.

Minor nit since this obviously works but something to think about

Reply:
sg, I have removed method and put code directly here